### PR TITLE
Add tutorials

### PR DESCRIPTION
**Pull request by CunliangGeng**
❗ Please first review this PR and then PR #17 .

Added tutorials on how to start a container of GameBus FHIR layer and how to send request to FHIR API.

Here is how to build html pages for the docs:
```bash
cd docs

# build the html pages
make html

# check html docs in browser 
make check

# remove html pages
make clean
```

### CODE REVIEW

File: docs/source/tutorial4_fhir_api.rst [hunk 1]
```diff
@@ -0,0 +1,187 @@
+Request on FHIR API
+===================
+
+After :ref:`starting FHIR server<tutorial2_docker_run:start fhir server>` and
+:ref:`adding data to GameBus platform<tutorial3_gamebus:add data manually>` in
+previous sections, now it's ready to try the service of GameBus FHIR layer.
+
+The FHIR layer is a service layer on GameBus platform, providing FHIR REST API
+for communicating FHIR-compliant data with outside. For example, when a user sends
```
Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:9]:
```suggestion
for communicating FHIR-compliant data with the outside. For example, when a user sends
```

File: docs/source/tutorial4_fhir_api.rst [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+an HTTP GET request to FHIR REST API, the FHIR layer will transform this request and
+forward the transformed request to GameBus's API; GameBus will process the request,
+e.g. reading requested data from its database, and then return the relevant data in
```
Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:12]:
```suggestion
e.g. read the requested data from its database, and then return the relevant data in
```

File: docs/source/tutorial4_fhir_api.rst [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+GameBus format to the FHIR layer; the FHIR layer will then transform the GameBus data to
+data in FHIR-compliant format, and then return the FHIR data to user as a HTTP
+response. That is how FHIR layer works and how it enables GameBus to provide FHIR
```
Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:15]:
```suggestion
response. That is how the FHIR layer works and how it enables GameBus to provide FHIR
```

Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:15]:
```suggestion
response. That is how FHIR layer works and how it enables GameBus to provide FHIR data.
```

File: docs/source/tutorial4_fhir_api.rst [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+service.
```
Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:16]:
```suggestion

```

File: docs/source/tutorial4_fhir_api.rst [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+
+To try and test FHIR API, we need API clients to help send HTTP requests.
+Various API clients exist, e.g. `httpie`_, `Postman`_ and `Hoppscotch`_.
```
Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:19]: I'd say curl can also be an option right?

File: docs/source/tutorial4_fhir_api.rst [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+   replace :code:`[pid]` with your player id.
+2. Click `Authorization` button
+3. Select authorization type :code:`Bearer`
+4. Fill in the token box with your GameBus token
+5. Send the request
+
+.. image:: image/fhir_api_patient.png
+  :width: 500
+
+The response is FHIR `Patient`_ resource, which is transformed from the
```
Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:78]:
```suggestion
The response is a FHIR `Patient`_ resource, which is transformed from the
```

File: docs/source/tutorial4_fhir_api.rst [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+1. Fill URL box with :code:`http://localhost:8080/Observation/[activity_id]`.
+   You need to replace :code:`[activity id]` with GameBus activity id, e.g.
+   walk activity id is :code:`790972`.
+2. Fill in Bearer token if it's empty
+3. Send the request
+
+.. image:: image/fhir_api_observation.png
+  :width: 500
+
+The response is FHIR `Observation`_ resource. Check the detail of the response body
```
Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:107]:
```suggestion
The response is a FHIR `Observation`_ resource. Check the detail of the response body
```

File: docs/source/tutorial4_fhir_api.rst [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+to see if the information is consistent with the activity data you added to GameBus.
+
+
+Search FHIR Observation resources
+---------------------------------
+
+Besides requesting FHIR observation with a specific id, FHIR layer also supports
```
Comment (dsmits) [inline docs/source/tutorial4_fhir_api.rst:114]:
```suggestion
Besides requesting FHIR observations with a specific id, FHIR layer also supports
```

**Review by dsmits — CHANGES_REQUESTED** [review]
Very nice. I have a couple of suggestions for the text.

File: docs/source/tutorial4_fhir_api.rst (hunk 5 above)
Comment (CunliangGeng) [reply docs/source/tutorial4_fhir_api.rst:19]: certainly

File: docs/source/index.rst [hunk 9]
```diff
@@ -0,0 +1,56 @@
+FHIR for GameBus
+================
+
+`GameBus`_ is a digital platform for people playing together healthy social,
```
[the remark this answers is not in the document]
Comment (CunliangGeng) [reply docs/source/index.rst]: Changed with some adaption. GameBus looks more family-oriented, so I added "family" before team and friends :-)

File: docs/source/tutorial1_install.rst [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+Hardware
+--------
+
+Mac with Apple silicon (e.g. M1, M2 cpu) is not supported
+
+
+Installation
+------------
+
+- `Chrome browser`_ (needed for `Chrome DevTools`_)
```
[the remark this answers is not in the document]
Comment (CunliangGeng) [reply docs/source/tutorial1_install.rst]: Chrome DevTools will be installed together with Chrome browser. It's not necessary to list DevTools as another item.

File: docs/source/tutorial2_docker_run.rst [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+To start the server, type the command below in a terminal:
+
+.. code-block:: bash
+
+  docker run -p 8080:8080 nlesc/gamebus-fhir-layer start_fhir_server https://api3-new.gamebus.eu/v2
+
+This command runs a Docker container using the :code:`nlesc/gamebus-fhir-layer`
+image,
+
+- :code:`-p 8080:8080` binds the container's port :code:`8080` to host machine's port :code:`8080`,
```
[the remark this answers is not in the document]
Comment (CunliangGeng) [reply docs/source/tutorial2_docker_run.rst:22]: Some of your comments are not shown on Github, and I accidently saw them in VSCode, e.g. this comment.

**Review by dsmits — CHANGES_REQUESTED** [review]
Does this help getting my comments to show?

**Review by dsmits — APPROVED** [review]
Looks good to me

**Comment by CunliangGeng** [comment]
> Does this help getting my comments to show?

No, I still cannot see the comment `Always nice if the components of the command are explained :+1:`

File: docs/source/tutorial2_docker_run.rst [hunk 12]
```diff
@@ -0,0 +1,38 @@
+Start FHIR server
+=================
+
+GameBus FHIR layer is `open-source`_ and available as a `Docker image`_
+(id: :code:`nlesc/gamebus-fhir-layer`). The most convenient way to deploy or run
+it is using Docker.
+
+This tutorial will show you how to run GameBus FHIR layer using Docker locally.
+
+Run Docker container
```
Comment (dsmits) [inline docs/source/tutorial2_docker_run.rst:10]: Good title

File: docs/source/tutorial2_docker_run.rst [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+--------------------
+
+To start the server, type the command below in a terminal:
+
+.. code-block:: bash
+
+  docker run -p 8080:8080 nlesc/gamebus-fhir-layer start_fhir_server https://api3-new.gamebus.eu/v2
```
Comment (dsmits) [inline docs/source/tutorial2_docker_run.rst:17]: good command

**Review by dsmits — COMMENTED** [review]
pointless review

File: docs/source/index.rst [hunk 14]
```diff
@@ -0,0 +1,56 @@
+FHIR for GameBus
+================
+
+`GameBus`_ is a digital platform where you can, together with your family, friends
```
Comment (dsmits) [inline docs/source/index.rst:4]: some comment

**Pull request merged**